pycaret/time_series/forecasting/oop.py

```diff
@@ -1242,6 +1248,15 @@ def setup(
                 "bfill", "pad", "ffill", "random"
             If int or float, imputation method is set to "constant" with the given value.
 
+        limit_target: Optional[List[Optional[float]]], default = None
```

Comment:
You will need to add the same argument to the functional API py file as well (in the same folder as oop.py)